numpy/compat/py3k.py

```diff
@@ -80,30 +80,6 @@ def is_pathlib_path(obj):
     """
     return isinstance(obj, Path)
 
-# from Python 3.7
-class contextlib_nullcontext:
```

Comment:
I haven't really found much usage of this outside of numpy, therefore suggest to remove it. If it gets the 👍, I think this is the only point that will need a changelog.